lib/fakeroot.py

```diff
@@ -209,8 +209,10 @@
    #      | egrep '^(fakeroot|fakeroot-ng|pseudo)$'
 
    "debSB":
-   { "name": "Debian 9 (Stretch) or 10 (Buster)",
-     "match": ("/etc/debian_version", r"^(9|10)\."),
+   { "name":   ("Debian based ditribution from following list:\n"
+                "Debian 9 (Stretch) or 10 (Buster)\n"
+                "Ubuntu 16 (Xenial), 18 (Bionic), or 20 (Focal)"),
```

Comment:
Can you:
1. adjust the key to reflect the new Ubuntu inclusion, and
2. revise the name to meet the character limit (see long tedious comment at the beginning of `DEFAULT_CONFIGS`